R/class.R

```diff
@@ -38,21 +38,21 @@
 #' @examples
 #' library(dplyr)
 #' library(modeldata)
-#' data(okc)
+#' data(Sacramento)
 #'
 #' # Learn the classes on the train set
-#' train <- okc[1:1000, ]
-#' test <- okc[1001:2000, ]
-#' recipe(train, age ~ .) %>%
+#' train <- Sacramento[1:500, ]
+#' test <- Sacramento[500:nrow(Sacramento), ]
```

Comment:
We don't want any overlap 😄 

```suggestion
#' test <- Sacramento[501:nrow(Sacramento), ]
```